runtime/testapps/src/main/resources/com/google/apphosting/runtime/jetty9/senderrorapp/ee8/hello.html

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
all html: need html syntax for comments, not Java /* style

Reply:
Done